validphys2/src/validphys/convolution.py

```diff
@@ -70,6 +70,25 @@ def _asy(a, b):
     return (a - b) / (a + b)
 
 
+def _addp(a, b):
+    """Compute positivity constraints for polarized gluon. The unpolarized
+    and polarized predictions have to follow a particular order:
+
+    Parameters:
+    -----------
+    a: tf.tensor
```

Comment:
This should work with any type of array, not only tf (and indeed, it does, otherwise vp will not be able to use it!)